app/controllers/admin/community_memberships_controller.rb

```diff
@@ -17,7 +17,7 @@ def index
       format.csv do
         all_memberships = CommunityMembership.where(:community_id => @community.id)
                                               .where("status != 'deleted_user'")
-                                              .includes(:person => :emails)
```

Comment:
Use `joins` instead of `includes`. As I understand it, it generates more queries, but allows more effective cache usage and in this case, more effective ActiveRecord instantiation. On large marketplaces lowers the query time from > 20 seconds to ~10 seconds.